tests/autocratV0.ts

```diff
@@ -1311,39 +1311,34 @@ describe("autocrat_v0", async function () {
       instruction;
 
     beforeEach(async function () {
-      await mintToOverride(context, mertdTreasuryMertdAccount, 1_000_000_000n);
-      await mintToOverride(context, mertdTreasuryUsdcAccount, 1_000_000n);
-
-      let receiver = Keypair.generate();
-      let to0 = await createAccount(
-        banksClient,
-        payer,
-        MERTD,
-        receiver.publicKey
-      );
-      let to1 = await createAccount(
-        banksClient,
-        payer,
-        USDC,
-        receiver.publicKey
-      );
-
-      const ix = await migrator.methods
-        .multiTransfer2()
-        .accounts({
-          authority: mertdDaoTreasury,
-          from0: mertdTreasuryMertdAccount,
-          to0,
-          from1: mertdTreasuryUsdcAccount,
-          to1,
-          lamportReceiver: receiver.publicKey,
-        })
-        .instruction();
-
+      const accounts = [
+        {
+          pubkey: mertdDao,
```

Comment:
Any reason this is mertDao?

Reply:
Think it's just like that to give some flavour to the tests